distribution/docker/src/docker/Dockerfile

```diff
@@ -14,7 +14,7 @@
 FROM ${base_image} AS builder
 
 RUN for iter in {1..10}; do yum update --setopt=tsflags=nodocs -y && \
-    yum install --setopt=tsflags=nodocs -y gzip shadow-utils tar && \
+    yum install --setopt=tsflags=nodocs -y epel-release && yum install --setopt=tsflags=nodocs -y dpkg wget gzip shadow-utils tar && \
```

Comment:
Rather than installing a Debian packaging tool, let's replace the `tini_suffix` build variable with `tini_arch`, and populate it with `amd64` or `arm64` as appropriate.